Dockerfile

```diff
@@ -72,16 +72,6 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
-# Install lxc
```

Comment:
we can put both changes in the same PR to merge them at the same time, although it's unusual.